modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/application/ApplicationResource.java

```diff
@@ -128,6 +129,29 @@ public ListApplicationJson list( @QueryParam("query") final String query )
     }
 
     @GET
+    @Path("listKeys")
+    public List<ApplicationKeyJson> listKeys( @QueryParam("query") final String query )
```

Comment:
No need to create the ApplicationKeyJson class for just one string field. Use List<String> instead. The method is named listKeys, so it is not expected to return more details, just the keys as string. And it is anyway untyped since it is sent as json.